mantis/blackbox/cvm_indexer.py

```diff
@@ -2,6 +2,7 @@
 
 from typing import List, Optional
 
+from custom_logging import logger
```

Comment:
will not just using loguru directly work? not abstraction needed.

Reply:
This abstruction for extracting logs from uvicorn.
But we can use directly work if so

Reply:
Switched to use original loguru fuctionality. But for common logging config need to implement 
https://github.com/erezinman/loguru-config

Reply:
yeah, do not delete it, thank you. may be will be needed at some. when will get heavy load on server.